internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -324,12 +324,13 @@ type image struct {
 }
 
 type Input struct {
-	Attestations []attestationData `json:"attestations"`
-	Image        image             `json:"image"`
+	Attestations  []attestationData       `json:"attestations"`
+	Image         image                   `json:"image"`
+	AllComponents []app.SnapshotComponent `json:"allcomponents"`
```

Comment:
I think some camel case would be more conventional here:

```suggestion
	AllComponents []app.SnapshotComponent `json:"allComponents"`
```